python/django/security/sql_injection_extra.yaml

```diff
@@ -0,0 +1,48 @@
+rules:
+  - id: sql-injection-using-extra-where
+    patterns:
+      - pattern-inside: |
+          def $F(...):
+            ...
+      - pattern-either:
+        # match when `format` is used
+        - pattern: |
+            $V = request.$W.get($X)
+            ...
+            $MODEL.objects.extra(..., where=[$S.format($V)], ...)
```

Comment:
I think doing `$S.format(..., $V, ...)` will catch more instances

Reply:
addressed. lmk what you think. Also, there where clause expects a list, but i cannot do 
```
where=[..., $S.format($V), ...]
```
in the list at the moment in sgrep